modules/advangelistsBidAdapter.js

```diff
@@ -0,0 +1,443 @@
+        console.log('About to create final respsone for banner')
+        return {
+          requestId: response.id,
+          bidderCode: BIDDER_CODE,
+          cpm: response.seatbid[0].bid[0].price,
+          width: response.seatbid[0].bid[0].w,
+          height: response.seatbid[0].bid[0].h,
+          ad: response.seatbid[0].bid[0].adm,
+          ttl: response.seatbid[0].bid[0].ttl || 60,
+          creativeId: response.seatbid[0].bid[0].crid,
+          currency: response.cur,
+          mediaType: BANNER,
+          netRevenue: true
+        }
+      }
+    }
+  }
+/*,
+
+  getUserSyncs(syncOptions, serverResponses = [], gdprConsent = {}) {
+    let syncs = [];
+    let { gdprApplies, consentString } = gdprConsent;
+    let bannerResponse = find(serverResponses, (res) => utils.isArray(res.body));
+
+    if (bannerResponse) {
+      if (syncOptions.iframeEnabled) {
+        bannerResponse.body
+          .filter(bid => bid.sync)
+          .forEach(bid => {
+            syncs.push({
+              type: 'iframe',
+              url: bid.sync
+            });
+          });
+      }
+    } else if (syncOptions.iframeEnabled) {
+      syncs.push({
+        type: 'iframe',
+        url: `//sync.bfmio.com/sync_iframe?ifg=1&id=${appId}&gdpr=${gdprApplies ? 1 : 0}&gc=${consentString || ''}&gce=1`
+      });
+    } else if (syncOptions.pixelEnabled) {
+      syncs.push({
+        type: 'image',
+        url: `//sync.bfmio.com/syncb?pid=144&id=${appId}&gdpr=${gdprApplies ? 1 : 0}&gc=${consentString || ''}&gce=1`
+      });
+    }
+
+    return syncs;
+  } */
+};
+
+function isBannerBid(bid) {
+  return utils.deepAccess(bid, 'mediaTypes.banner') || !isVideoBid(bid);
+}
+
+function isVideoBid(bid) {
+  return utils.deepAccess(bid, 'mediaTypes.video');
+}
+
+function isVideoBidValid(bid) {
+  return isVideoBid(bid) && getVideoBidParam(bid, 'pubid') && getVideoBidParam(bid, 'placement');
+}
+
+function isBannerBidValid(bid) {
+  return isBannerBid(bid) && getBannerBidParam(bid, 'pubid') && getBannerBidParam(bid, 'placement');
+}
+
+function getVideoBidParam(bid, key) {
+  return utils.deepAccess(bid, 'params.video.' + key) || utils.deepAccess(bid, 'params.' + key);
+}
+
+function getBannerBidParam(bid, key) {
+  return utils.deepAccess(bid, 'params.banner.' + key) || utils.deepAccess(bid, 'params.' + key);
+}
+
+function isMobile() {
+  return (/(ios|ipod|ipad|iphone|android)/i).test(navigator.userAgent);
+}
+
+function isConnectedTV() {
+  return (/(smart[-]?tv|hbbtv|appletv|googletv|hdmi|netcast\.tv|viera|nettv|roku|\bdtv\b|sonydtv|inettvbrowser|\btv\b)/i).test(navigator.userAgent);
+}
+
+function getDoNotTrack() {
+  return navigator.doNotTrack === '1' || window.doNotTrack === '1' || navigator.msDoNoTrack === '1' || navigator.doNotTrack === 'yes';
+}
+
+function findAndFillParam(o, key, value) {
+  try {
+    if (typeof value === 'function') {
+      o[key] = value();
+    } else {
+      o[key] = value;
+    }
+  } catch (ex) {}
+}
+
+/*
```

Comment:
Same as above, please remove commented out code block. I'd recommend creating a branch in your fork with the commented out code blocks if you need to use for development/testing.

Reply:
Thanks. Have removed commented block.